lib/repl.js

```diff
@@ -1136,15 +1125,14 @@ function complete(line, callback) {
                 }
               }
             } catch (e) {
-              //console.log("completion error walking prototype chain:" + e);
+              // console.log("completion error walking prototype chain:" + e);
             }
           }
 
           if (memberGroups.length) {
             for (i = 0; i < memberGroups.length; i++) {
-              completionGroups.push(memberGroups[i].map(function(member) {
-                return expr + '.' + member;
-              }));
+              completionGroups.push(
+                memberGroups[i].map((member) => expr + '.' + member));
             }
             if (filter) {
               filter = expr + '.' + filter;
```

Comment:
filter = expr + '.' + filter;
This can be changed to 
filter = `${ expr }.${ filter }`;
Since it removes access usage of +

Reply:
Done